src/Http/GraphRequest.php

```diff
@@ -401,7 +399,7 @@ function ($reason) {
     * @param string $path   The path to download the file to
     * @param mixed  $client The client to use in the request
     *
-     * @throws GraphException if file path is invalid; if \GuzzleHttp\Exception\BadResponseException is thrown for 4xx/5xx responses
+     * @throws GraphException if file path is invalid
      *
```

Comment:
I think we can add an additional `@throws` for BadResponseException since we handle that on line 435?

Reply:
Not sure. Actually the functions `execute()`, `download()` and `upload()` all can potentially throw any GuzzleException (including but not limited to BadResponseException), and we don't really "handle" it (only un-truncate the message)... Maybe add `@throws GuzzleException` then?

Reply:
Good point. `@throws GuzzleException` makes sense. And yes sorry, we don't "handle" it, wrong word 😄